azure-pipelines.yml

```diff
@@ -0,0 +1,157 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/3.0
```

Comment:
What is the plan for branching going forward? Should we include a wildcard so that infra is ready for next releases' branching?

Reply:
current release is being shipped from the main branch. Release cadance of the repo makes it so that we have stable releases every month or every other month, depending on the incoming changes. Due to this, having separate branches becomes more of a burden than helping stabilize, so we have currently opted to just continue shipping out of main for now, and for larger things having feature branches until code is mature enough to be merged on main. All that said, I'm fine with deciding to add a wildcard here, I currently just left this as it is on our other pipeline.